core/experiments/manager.php

```diff
@@ -993,4 +952,104 @@ public function __construct() {
 			\WP_CLI::add_command( 'elementor experiments', WP_CLI::class );
 		}
 	}
+
+	/**
+	 * @param array $experimental_data
+	 * @return array
+	 * @throws Dependency_Exception
+	 */
+	private function check_dependencies( array $experimental_data ): array {
```

Comment:
```suggestion
	private function validate_and_initialize_dependencies( array $experimental_data ): array {
```
WDYT? I am not sure about the name `check_dependencies`, since it also modifies the input array.